From a1b9bb4af819ed389675f16e4a521efeda4cc3f3 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sat, 27 Mar 2021 22:10:48 -0700
Subject: [PATCH] do not export the CHARS_PER_LINE variable

The library exports a variable named "CHARS_PER_LINE". This is a generic name that could conflict with a name in user's code.
Please either rename the variable or make it static.

Upstream-Status: Submitted [http://sourceforge.net/tracker/?func=detail&aid=3591420&group_id=152942&atid=785907]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 src/cencode.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/cencode.c b/src/cencode.c
index 03ba5b6..3df62a8 100644
--- a/src/cencode.c
+++ b/src/cencode.c
@@ -7,7 +7,7 @@ For details, see http://sourceforge.net/projects/libb64
 
 #include <b64/cencode.h>
 
-const int CHARS_PER_LINE = 72;
+static const int CHARS_PER_LINE = 72;
 
 void base64_init_encodestate(base64_encodestate* state_in)
 {
